src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetPersonServiceTest.kt

```diff
@@ -62,7 +65,10 @@ internal class GetPersonServiceTest(
       whenever(probationOffenderSearchGateway.getPerson(id = hmppsId)).thenReturn(Response(data = null))
```

Comment:
I don't think there's a situation where data would be null on return of the getPersonsService